src/config/main.js

```diff
@@ -48,13 +49,15 @@ const defaultBlockSettings = {
 	photodiode_active: false
 }
 
+// setting config for trials
+const config = init({USE_PHOTODIODE: !AT_HOME && IS_ELECTRON,  USE_EEG: false, IS_ELECTRON: IS_ELECTRON, USE_MTURK: MTURK})
```

Comment:
`IS_ELECTRON: IS_ELECTRON` can be destructured into `IS_ELECTRON`

Reply:
but should it be USE_ELECTRON in the behavioral task library to be more consistent with the others?

Reply:
Yes!

Reply:
this one still needs to be resolved